website/src/content/docs/reference/configuration.mdx

```diff
@@ -451,6 +451,18 @@ How many characters can be written on a single line.
 
 > Default: `80`
 
+### `formatter.bracketSameLine`
+
+Choose whether the ending `>` of a multi-line HTML ( or HTML-like ) element should be on the last attribute line or not
```

Comment:
That's not HTML but JSX. Biome doesn't support html yet, so it would be misleading

Reply:
That's right sorry for the misunderstanding, fixed that!